examples/binderhub/deploy.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
Maybe delete/separate the fadi installation part form this script? If we let the fadi installation, we will need to modify the genral bash script and this one.

Reply:
Ok, I deleted the part concerning FADI and explain into the README that there are two scripts.